zokrates_core/src/lib.rs

```diff
@@ -29,7 +29,7 @@ extern crate zokrates_pest_ast;
 mod embed;
 mod flatten;
 mod helpers;
-mod imports;
+pub mod imports;
```

Comment:
@dark64 I guess this is needed for zokrates.js?

Reply:
Yes, we use "zokrates_core::imports::Error" to properly return resolving errors (e.g. https://github.com/blockchain-it-hr/zokrates-js/blob/master/src/lib.rs#L36)